src/lib/config/nav-menu/tool-selector-nav-items.ts

```diff
@@ -47,30 +47,42 @@ export const toolSelectorNavItems = {
                             label: "Topcoder Academy",
                             url: "http://platform-ui.topcoder.com/learn",
                             icon: "tcacademy",
-                            description: "Hover description text"
+                            description: "Learn new skills"
                         },
                         {
                             label:"Dev Center",
                             icon: "dev-center",
                             url: "http://platform-ui.topcoder.com/dev-center",
-                            description: "Hover description text"
+                            description: "Get help for Topcoder tools"
+                        },
+                        {
+                            label:"Practice",
+                            // icon: "practice",
```

Comment:
Is this a placeholder while we wait on UX assets?

Reply:
We don't have an icon for "practice" in figma right now. It will render the icon placeholder if we don't send an Icon.

Reply:
Ok